pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -90,6 +90,9 @@ Examples:
 			checkErr(err)
 			newRc := obj.(*api.ReplicationController)
 
+			if len(namespace) == 0 {
+				namespace = api.NamespaceDefault
```

Comment:
Hrm - this is not a decision this code should make.  Was there a config file with an empty namespace in the call path?